tests/integration/README.md

```diff
@@ -0,0 +1,34 @@
+# Maistra Istio Operator Integration Test
+
+This integration test suite is similar to the upstream Istio integration tests. The scripts `lib.sh`, `kind_provisioner.sh` and `istio-integ-suite-kind.sh` are copied from `github.com/maistra/istio` repo.
+
+## How to Run it Manually
+
+1. Create a builder container:
+
+```
+$ docker pull quay.io/maistra-dev/maistra-builder:2.3
+$ docker run  -d -t --name test --privileged \
+    -e BUILD_WITH_CONTAINER="0" \
+    -v /lib/modules:/lib/modules \
+    -v /sys/fs/cgroup:/sys/fs/cgroup \
+    --mount type=tmpfs,destination=/var/lib/docker \
+    -m 4Gi \
+    --cpus="4" \
+    quay.io/maistra-dev/maistra-builder:2.3 \
+    entrypoint tail -f /dev/null
+```
```

Comment:
I think this command could be simplified to sth like
```
docker run -it --rm --privileged -v /var/lib/docker -v $PWD:/work --entrypoint entrypoint quay.io/maistra-dev/maistra-builder:2.3 bash
```

This will give you a shell inside the container. You can run the `make test.integration.kind` command in there, or, if you don't need a shell you can replace the `bash` command above with `make test.integration.kind`.

Reply:
Hi Jonh, the reason why I use entrypoint `tail -f /dev/null` is for running a container first and then from host i can copy the source code using `docker cp . test:/work`

Reply:
well, I don't think we should mount $PWD:/work. That will mess up the host istio-operator directory permissions.
Because we run as root in a builder container and there is a `make image` command in building the operator image locally. If we mount it, the host istio-operator directory permission will be changed to root. 

I think entrypoint tail -f /dev/null and then run docker cp . test:/work  is a safer way of trouble shooting locally.